### ISSUE TITLE
Create a new server description and update all server lists

### ISSUE THREAD

**Issue by uncovery** [uncovery/uncovery_me#163]

**Comment by psiber-on-uncovery** [comment]
Had a play with putting together a punchier login icon:

![server-icon-recovered](https://cloud.githubusercontent.com/assets/16591000/12533949/5bfbee7c-c27f-11e5-859a-55060ac848a7.png)

Minecraftservers.org current description
>Online since Oct 2010
>100% Free, 247 prof. hosted server
>Stable mature community of all ages
>Protected 128x128 survival or creative lots
>Griefer & idiot free
>Working economy
>skyblock, aether, endless worlds
>No overpowered mods, no handouts, no cheating
>IRC, Teamspeak, 2D & 3D Maps, active blog
>many updates and dedicated builders
>Whitelist on the website!

Draft description
> Online, active server and users since 2010
> Griefer & idiot free
> Free, not pay to win economy server
> Protected, high grade plot system. Survival / Creative options.
> Professionally hosted, high uptime 24/7 server
> Many world types to explore and claim plots in.
> IRC, Teamspeak, 2D & 3D Maps, active blog
> Daily updates and dedicated builders
> No overpowered mods, no handouts, no cheating
> Whitelist at http://uncovery.me

**Comment by psiber-on-uncovery** [comment]
minecraft-server-list.org description current

Online since Oct 2010, Free, 247 prof hosted server with a stable mature community. Protected 128x128 survival or creative lots for everyone. Griefer idiot free. Contests, custom shop system, custom voting lottery, custom adventure hunger games systems.

No overpowered mods, no handouts, no cheating. IRC, Teamspeak, 2D 3D Maps, active blog, many updates and dedicated builders.
We disabled comments below since many users would post there asking to be whitelisted, instead of reading this here. Since you are reading this, you are obviously smarter than them those people and you just found out that you need to

PLEASE WHITELIST ON THE WEBSITE, click visit Website -

**Comment by psiber-on-uncovery** [comment]
planet minecraft description current

Whitelist yourself on the website:
http://uncovery.me
Advanced Server for advanced minecrafters n
- Survival Minecrafting at its finest n
- Get a 128x128 protected lot n
- Instant whitelist & buildrights applications n
- Griefer-safe, frequent updates of plugins n
- good community, friendly admins, server live since October 2010
- 24/7 prof hosted 8GB lag-free environment n
- Blog, Maps .w user locations, IRC, Teamspeak, Steam Group n
- Multi-world

**Comment by psiber-on-uncovery** [comment]
minestatus.net

Advanced Server for advanced minecrafters
- Free, dedicated server, 24/7, lagfree
- Survival with multiple worlds, 128x128 lots for all
- Griefer-safe, frequent updates of plugins
- good community, friendly admins, server live since October 2010
- Minecart Mania, Worldguard, Towny

very outdated description.

**Comment by psiber-on-uncovery** [comment]
http://minecraft-mp.com/server-s49 

Survival & Creative worlds
Nether, End, Skylands, Flatlands & others
24/7 Lag free, 50 Slots
100% FREE
Griefer- & Idiot free
since October 2010
Bukkit
Great builders & community
reserve 128x128 lots towny-worlds

outdated again :(

**Comment by psiber-on-uncovery** [comment]
Ok, so generally speaking all the lists give a good amount of space for description.

Maybe we try something like

UNCOVERY - Whitelist and come play with us! (call to action)
http://uncovery.me

Playing here we guarantee you will:
> have heaps of protected space to build and gather!
> never have issues with lag, grief or trolls!
> never pay to win. Perks are earned not bought!
> always be able to login, high up-time 247 server!
> be able to sell goods in the player shop to players!
> enjoy exploring worlds including aether, darklands, skyblock ...
> be able to win great rewards in the vote lottery!
> discover why people have played here for 5+ years!
> have access to creative worlds, survival worlds and hardcore worlds!

Thoughts?

**Comment by darthandrew1** [comment]
UNCOVERY - Be yourself! Build YOUR Dream!
http://www.uncovery.me

```
- A friendly, whitelisted, lot-based community

- Everyone is included in the fun!

- No lag, griefing, or trolls

- 24/7 uptime

- NO Pay-to-Win. It's all fair here!

- Vote to win great rewards from our lottery!

- Start off with either creative or survival lots (You choose!)

- Buy and sell goods with our custom shop
```

Read more about our server at http://www.uncovery.me - Have fun

**Comment by uncovery** [comment]
I would love to do this a bit more systematic than discussing wording right away.
So let's first list what features we think are relevant to show to users:

IMO:

- server up since 2010
- no griefing / trolls
- lag free
- play-to-win instead of pay-to-win
- multiverse (creative, survival, skyblock)
- big builds
- working economy

Things I like to avoid to mention:
- custom plugins (that normally means it does not work)
- voting 

Things we can mention but not sure
- all the chat options such as teamspeak

**Comment by darthandrew1** [comment]
I've edited my post.

**Comment by dani0010** [comment]
The maturity (actual adulthood!) and longevity of our elders and masters as contrasted with the rotating gangs of children who work as moderators on other servers is huge. I'll have to consider how to distill that into a single pithy phrase.

**Comment by uncovery** [comment]
@dani0010 good point. I wonder how to say that without sounding like an elite club or a closed community hard to break into. "stable community with average x months of membership" maybe?

**Comment by psiber-on-uncovery** [comment]
professionally moderated by non-children?

**Comment by uncovery** [comment]
hehe good one. maybe rather say "properly moderated" instead of professionally. We don't pay anyone finally.
